proposals/semi-auto-properties.md

```diff
@@ -12,114 +12,85 @@ In these cases by now you always have to create an instance field and write the
 
 ## Specification changes
 
-The following changes are to be made to [§14.7.4](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/classes.md#1474-automatically-implemented-properties):
+[§14.7.1](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1471-general) *Properties - General*
+
+> A *property_initializer* may only be given for ~~an automatically implemented property, and~~ **a property that has a backing field that will be emitted and the property either does not have a setter, or its setter is auto-implemented. The *property_initializer*** causes the initialization of the underlying field of such properties with the value given by the *expression*.
```

Comment:
i honestly ca'nt made heads or tails of this sentence.  there are too many ands, ors, and nots.  And, based on my simple reading of this, this would disallow allow:

```c#
strnig X { get => field; set => field = value.Trim() } = "foo";
```

From my read, this clause doesn't apply "does not have a setter".  nor does "its setter is auto-implemented" apply.  

But the above should work.

Can we break this up into clearer rules?

Reply:
My intent for the PR was to simply re-format the existing statements that are in `diff` blocks as *inline diffs* - that is, using ~~strike-through~~ for deletions and **bold* for additions.

We can certainly modify the existing statements, but I'd prefer to do that in a follow-up change.

Reply:
gotcha.